test/unit/mdc-slider/foundation.test.js

```diff
@@ -426,3 +427,57 @@ test('#setMin updates "aria-valuemin" to the new minimum', () => {
 
   raf.restore();
 });
+
+test('#isDisabled/#setDisabled retrieves / sets the disabled state, respectively', () => {
```

Comment:
I'd update this test name to be more explicit about what you're asserting. You're asserting that `isDisabled()` returns the disabled state as set by `setDisabled()` or something along those lines.

Reply:
Done!